demand_energy.md

```diff
@@ -203,4 +237,3 @@ Residential and commercial
 
 ## References
```

Comment:
I presume there's at least one paper to add as a reference?

Reply:
Added references with hyperlinks